dfastmi/io/RiverParameterData.py

```diff
@@ -49,25 +51,26 @@ class RiverParameterData:
     """ 
         The dictionary containing river data.
     """
-    _is_initialized : bool
+    #_is_initialized : bool
```

Comment:
whats the plan with this commented part?

Reply:
will be removed